### ISSUE TITLE
Better way to parse output of smartctl analysis command

### ISSUE THREAD

**Issue by sukhbir-singh** [sukhbir-singh/smart-status-plugin#1]
Right now, this plugin works well for smartctl version 6.5. But there exists the latest version of smartctl i.e. version 7 and it is default version in fedora 29.

So we need to update parsing of smartctl output. Here, the problem is with each new version parsing needs to be changed little bit by observing the smartctl output. We can deal with this problem by using --json flag that is introduced in latest version of smartctl and so using this parsing would be easier and more robust.

**Comment by cpg** [comment]
Can we do it in a backwards-compatible way?

**Comment by sukhbir-singh** [comment]
Yes. We can easily check version of smartctl installed and accordingly we can choose either to use standard parsing technique or json parsing.
Json flag option is only available in latest version 7 of smartctl.
